modyn/evaluator/internal/grpc/evaluator_grpc_servicer.py

```diff
@@ -76,9 +72,6 @@ def __init__(self, config: dict, tempdir: pathlib.Path):
         model_storage_address = f"{config['model_storage']['hostname']}:{config['model_storage']['port']}"
         self._model_storage_stub = EvaluatorGRPCServicer.connect_to_model_storage(model_storage_address)
 
-        self._selector_address = f"{config['selector']['hostname']}:{config['selector']['port']}"
-        self._selector_stub = EvaluatorGRPCServicer.connect_to_selector(self._selector_address)
-
```

Comment:
connection to selector is no longer needed since we won't evaluate on trigger training set